packages/digit_data_model/lib/data/oplog/oplog.dart

```diff
@@ -164,59 +164,59 @@ class ProjectOpLogManager extends OpLogManager<ProjectModel> {
 }
 
 //oplog1
-class ServiceDefinitionOpLogManager
-    extends OpLogManager<ServiceDefinitionModel> {
-  ServiceDefinitionOpLogManager(super.isar);
-
-  @override
-  ServiceDefinitionModel applyServerGeneratedIdToEntity(
-    ServiceDefinitionModel entity,
-    String serverGeneratedId,
-    int rowVersion,
-  ) =>
-      entity.copyWith(id: serverGeneratedId, rowVersion: rowVersion);
-
-  @override
-  String getClientReferenceId(ServiceDefinitionModel entity) {
-    throw UnimplementedError();
-  }
-
-  @override
-  String? getServerGeneratedId(ServiceDefinitionModel entity) => entity.id;
-
-  @override
-  int? getRowVersion(ServiceDefinitionModel entity) => entity.rowVersion;
-
-  @override
-  bool? getNonRecoverableError(ServiceDefinitionModel entity) =>
-      entity.nonRecoverableError;
-}
-
-//oplog2
-class ServiceOpLogManager extends OpLogManager<ServiceModel> {
-  ServiceOpLogManager(super.isar);
-
-  @override
-  ServiceModel applyServerGeneratedIdToEntity(
-    ServiceModel entity,
-    String serverGeneratedId,
-    int rowVersion,
-  ) =>
-      entity.copyWith(id: serverGeneratedId, rowVersion: rowVersion);
-
-  @override
-  String getClientReferenceId(ServiceModel entity) => entity.clientId;
-
-  @override
-  String? getServerGeneratedId(ServiceModel entity) => entity.id;
-
-  @override
-  int? getRowVersion(ServiceModel entity) => entity.rowVersion;
-
-  @override
-  bool? getNonRecoverableError(ServiceModel entity) =>
-      entity.nonRecoverableError;
-}
+// class ServiceDefinitionOpLogManager
```

Comment:
What is the reason of commenting this code

Reply:
> What is the reason of commenting this code

I moved this to checklist package, forgot to remove it from here.